### PR TITLE
Create start page for PII redaction APP

### PR DESCRIPTION
**Pull request by gabechu**
### Description
This PR adds the start (sidebar) html for the PII redaction app and this html page would get loaded to Support instance as an embedded document using `iframe`. The development is conducted locally, so `manifest.json` is updated with a local address for Flask app.

What are the changes in this PR

- Adding a route to `sidebar` -- index page of the app
- Testing `sidebar` route
- Run unittest for `app_remote` folder with travis
- Run flake8 for `app_remote` folder with travis

#### Checklist
- [x] Added **tests** for changed code.

### CODE REVIEW

File: app_remote/tests/test_sidebar.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+def test_sidebar(client):
+    actual = client.get("/sidebar")
+    assert actual.status_code == 200
+    assert actual.data is not None
```
Comment (torstenrudolf) [inline app_remote/tests/test_sidebar.py:4]: why not test the actual content?

**Comment by torstenrudolf** [comment]
wondering why the projects are called `app_local` and `app_remote`. could we find better names? seems like one is the backend and the other the frontend?

File: app_remote/tests/test_sidebar.py (hunk 1 above)
Comment (gabechu) [reply app_remote/tests/test_sidebar.py:4]: Content will change soon as we are going to add a `redaction` button and possibly more. Testing actual data may require us to constantly update the test reflecting the change we made. Just wondered is it a convention for people to test the actual content in web development?

**Comment by gabechu** [comment]
> wondering why the projects are called `app_local` and `app_remote`. could we find better names? seems like one is the backend and the other the frontend?

These names are borrowed from Zendesk tutorials where they use `app_local` referring a ZAT server runs locally and `remote` is the address hosting everything.

I do think `frontend` and `backend` would make more sense. One reason stops me from doing it is I'm using templating in Flask (at least it'll be for the first iteration of the app), which isn't a strict fit of the definition of backend.

File: app_remote/tests/test_sidebar.py (hunk 1 above)
Comment (torstenrudolf) [reply app_remote/tests/test_sidebar.py:4]: I think it's good practise to check the response body. maybe just look for sub-strings and don't check the whole html text, in order to minimise updates when changes occur. But it's good to at least smoke test to make sure that this endpoint doesn't return garbage.

**Pull request merged**